src/core/core.scale.js

```diff
@@ -610,8 +609,24 @@ var Scale = Element.extend({
 	},
 
 	/**
-	* @private
-	*/
+	 * @private
+	 */
+	_getLabelSizes: function() {
+		var me = this;
+		var labelSizes;
+
+		if (me._labelSizes) {
+			return me._labelSizes;
+		}
+
+		me._labelSizes = labelSizes = computeLabelSizes(me.ctx, parseTickFontOptions(me.options.ticks), me.getTicks(), me.longestTextCache);
+		me.longestLabelWidth = labelSizes.widest.width;
+		return labelSizes;
```

Comment:
Does this help to save the size?
```js
var labelSizes = me._labelSizes;

if (!labelSizes) {
    // ...
}
return labelSizes;
```

Reply:
Yes, that's smaller. I've updated it to that